### DISCUSSION TITLE
Remove the Walrus

### DISCUSSION THREAD

**Discussion by Kami-Interface** [Team-QUIET/QUIET-Community-Edition#129] [Ideas]
I don't think every faction needs to be able to traverse the water with a complete platoon without the usage of transports. If we look at the vanilla units UEF has very few amphibious units, and that has always been one of their weaknesses. With LOUD there have been MANY unit introductions to fill these gaps. I think gaps in a faction's portfolio add to its uniqueness and help them stand apart more.
If there is an issue with the Demolisher I'd see those get resolved, and perhaps the demolished buffed a bit if it's too weak compared to the other factions mobile artillery.

**Comment by Azraeel** [comment]
I really think I couldn't agree more with everything said. 
Weaknesses in a faction and the mismatches it makes, make this game far better.
Every faction having the same abilities makes the factions all just reskins and essentially eliminates most of the diversity.

**Comment by PurpleMossClump** [comment]
I just don't see the need for the UEF to have a unit of every class that can traverse water.  The UEF already has what can easily be argued to be the best T3 transport in the game, the Continental. The thing was designed specifically for them to be able to land full armies wherever they wanted to get them. 

And, while the Walrus is strictly a better Demolisher, having the same range and damage but a faster rate of fire, that can easily just be tweaked on the Demolisher by increasing its fire rate.
